orangecontrib/educational/widgets/owpolynomialregression.py

```diff
@@ -64,6 +64,9 @@ class Error(OWWidget.Error):
         """
         all_none = Msg("One of the features has no defined values")
 
+    class Warning(OWWidget.Warning):
+        no_cont_variables = Msg("No continuous variables")
```

Comment:
This looks like an Error, if it breaks execution. Might also reword into something nicer.

Reply:
Done.